lib/linter/index.js

```diff
@@ -42,17 +42,29 @@ exports.lint = function () {
 
     return results.results.map((result) => {
 
-        return {
-            filename: result.filePath,
-            errors: result.messages.map((err) => {
+        const transformed = {
+            filename: result.filePath
+        };
+
+        if (result.hasOwnProperty('output')) {
+
+            transformed.fix = {
+                filePath: result.filePath,
```

Comment:
shouldn't need `filePath` set here, can use the `filename` property of `transformed`